rds.tf

```diff
@@ -32,7 +32,7 @@ resource "aws_rds_cluster" "db" {
   skip_final_snapshot             = var.skip_final_snapshot
   copy_tags_to_snapshot           = true
   vpc_security_group_ids          = var.vpc_security_group_ids
-  db_subnet_group_name            = aws_db_subnet_group.rds.id
+  db_subnet_group_name            = var.db_subnet_group_name != "" ? var.db_subnet_group_name : aws_db_subnet_group.rds[0].id
```

Comment:
create local.db_subnet_group_name
need to add subnet group name in aws_rds_cluster_instance resource